src/ch.usi.inf.nodeprof/js/analysis/expression/minitests.switch.js.expected

```diff
@@ -11,9 +11,9 @@ expression starts: (src/ch.usi.inf.nodeprof.test/js/minitests/switch.js:19:16:32
 expression finishes: (src/ch.usi.inf.nodeprof.test/js/minitests/switch.js:19:16:32:2) type: ClosureFunctionExpression
 expression starts: (src/ch.usi.inf.nodeprof.test/js/minitests/switch.js:20:11:20:12) type: JSReadCurrentFrameSlot
 expression finishes: (src/ch.usi.inf.nodeprof.test/js/minitests/switch.js:20:11:20:12) type: JSReadCurrentFrameSlot
-statement finishes: (src/ch.usi.inf.nodeprof.test/js/minitests/switch.js:20:11:20:12) stmt: JSWriteScopeFrameSlot
-expression starts: (src/ch.usi.inf.nodeprof.test/js/minitests/switch.js:20:11:20:12) type: JSReadScopeFrameSlot
-expression finishes: (src/ch.usi.inf.nodeprof.test/js/minitests/switch.js:20:11:20:12) type: JSReadScopeFrameSlot
+statement finishes: (src/ch.usi.inf.nodeprof.test/js/minitests/switch.js:20:11:20:12) stmt: JSWriteCurrentFrameSlot
+expression starts: (src/ch.usi.inf.nodeprof.test/js/minitests/switch.js:20:11:20:12) type: JSReadCurrentFrameSlot
+expression finishes: (src/ch.usi.inf.nodeprof.test/js/minitests/switch.js:20:11:20:12) type: JSReadCurrentFrameSlot
```

Comment:
I'm not really sure why we have the class names here (it's an impl detail in the end)

Reply:
This is basically for debugging. There's an old TODO note [here](https://github.com/Haiyang-Sun/nodeprof.js/blob/master/src/ch.usi.inf.nodeprof/js/analysis/trivial/emptyTemplate.js#L127) for translating the internal node names. Hasn't happened though.